chsdi/templates/htmlpopup/kgs_anhoerung.mako

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <% c['stable_id'] = True %>
+    <tr><td class="cell-left">${_('ch.babs.kulturgueter.zkob')}</td>   <td>${c['attributes']['zkob']}</td></tr>
+    <tr><td class="cell-left">${_('y')}</td>              <td>${int(round(c['attributes']['x'],0)) or '-'}</td></tr>
+    <tr><td class="cell-left">${_('x')}</td>              <td>${int(round(c['attributes']['y'],0)) or '-'}</td></tr>
+    <tr><td class="cell-left">${_('gemeinde')}</td>       <td>${c['attributes']['gemeinde'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('kanton')}</td>         <td>${c['attributes']['kt_kz'] or '-'}</td></tr>
+</%def>
+
+<%def name="extended_info(c, lang)">
+    <%
+        c['stable_id'] = True
+        objarts = c['attributes']['objektart'].split(',')
+        import csv
+        from urllib2 import urlopen
+        dataGeoAdminHost = request.registry.settings['datageoadminhost']
+        dataPath = 'ch.babs.kulturgueter'
+        csv_url = "https://" + dataGeoAdminHost + "/ch.babs.kulturgueter-anhoerung/image/meta.txt"
+        csv_file = None
+        try:
+            csv_file = urlopen(csv_url)
+            reader = csv.reader(csv_file, delimiter =';')  # creates the reader object
+            pic_list = []
+            for i, row in enumerate(reader):   # iterates the rows of the file in orders
+                if i == 0: # The first row is NUMMER;BILDNR;FOTOGRAF;COPYRIGHT and cannot be parsed.
+                    continue
+                if int(row[0]) == c['featureId']:
+                    pic_list.append(map(lambda x: x.decode('utf-8'), row))
+        finally:
+            csv_file.close()
+    %>
+    <script>
+        $(document).ready(function(){
+            $('.thumbnail-container').on('click', function (event) {
+              event = event || window.event;
+                event.preventDefault();
+              var target = event.target || event.srcElement,
+                link = target.src ? target.parentNode : target,
+                options = {index: link, event: event, onslide: function(index, slide){
+                    /** a "beautiful" line of code which sets the title of the gallery to the current copyright + photographer of the current photo**/
+                    $('#blueimp-gallery-title').html(($($('.thumbnail-container').children('.thumbnail')[index]).children('div').html()));
+                }},
+                links = this.getElementsByTagName('a');
+              blueimp.Gallery(links, options);
+            });
+        });
+    </script>
+
+
+    <table class="table-with-border kernkraftwerke-extended">
```

Comment:
`kernkraftwerke-extended` This is a funny class name. Is it possible to generalize this name if it is needed here?

Reply:
Not changing this in this pull request as it touches around 30 makos.